lib/sms.rb

```diff
@@ -0,0 +1,16 @@
+class Sms
+
+  def initialize
+    @tokens = YAML::load(File.open('./lib/resources/tokens.yml'))
+  end
+
+   def send(delivery_time)
+     @tokens = YAML::load(File.open('./lib/resources/tokens.yml'))
+     client = Twilio::REST::Client.new @tokens["account_sid"], @tokens["auth_token"]
+     client.account.messages.create(
+     :from => @tokens["from_number"],
+     :to => @tokens["to_number"],
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)